torch/distributed/tensor/placement_types.py

```diff
@@ -465,37 +446,35 @@ def _split_tensor(
         with_padding: bool = True,
         contiguous: bool = True,
     ) -> tuple[list[torch.Tensor], list[int]]:
-        """
```

Comment:
would be nice to document the intuition behind StridedShard._split_tensor.  IIUC it is 
 - we 'split' in 'Left to Right' order, meaning for [StridedShard(0, sf=2), Shard(0)] we first split the tensor over the strided placment, and then later we split it over the shard placement.
example:

orig tensor:
[0, 1, 2, 3, 4]

Step 1: Strided shard (including strided split and cat)
[0, 1] [2, 3] [4, ] []
[0, 1, 4],  [2, 3]

Step 2: Regular Shard(0) split
Rank0: [0, 1]   Rank1 [2,]
Rank2: [4,]     Rank3 [3]